lib/chat_api_web/controllers/registration_controller.ex

```diff
@@ -12,30 +12,29 @@ defmodule ChatApiWeb.RegistrationController do
     try do
       invite = ChatApi.UserInvitations.get_user_invitation!(user_params["invite_token"])
       params = Enum.into(user_params, %{"account_id" => invite.account.id})
-
       conn
       |> Pow.Plug.create_user(params)
       |> case do
-        {:ok, _user, conn} ->
-          if ChatApi.UserInvitations.expired?(invite) do
-            send_server_error(conn, "Invitation token has expired")
-          else
-            ChatApi.UserInvitations.expire_user_invitation(invite)
-            send_api_token(conn)
-          end
-
-        {:error, changeset, conn} ->
-          errors = Changeset.traverse_errors(changeset, &ErrorHelpers.translate_error/1)
-          send_user_create_errors(conn, errors)
-      end
+           {:ok, _user, conn} ->
```

Comment:
mind running `mix format` before committing? (i'm worried the diff is getting cluttered by autoformatting 😛 )

Reply:
Sure thing!